h/static/scripts/login-forms/components/test/LoginForm-test.js

```diff
@@ -116,6 +116,44 @@ describe('LoginForm', () => {
     assert.equal(passwordField.prop('value'), fakeConfig.formData.password);
   });
 
+  [
+    // Normal login page
+    {
+      prefillUsername: false,
+      usernameError: false,
+      autofocusUsername: true,
+    },
+    // Login page with prefilled username
```

Comment:
The username can be prefilled without error either because of a `username` query param, or after a login failure due to a password issue.